D/ducc0/build_tarballs.jl

```diff
@@ -6,7 +6,7 @@ using BinaryBuilderBase
 include(joinpath(@__DIR__, "..", "..", "platforms", "microarchitectures.jl"))
 
 name = "ducc0"
-version = v"0.29.0"
+version = v"0.29.1"
```

Comment:
Upstream version is the same
```suggestion
version = v"0.29.0"
```